cluster/addons/fluentd-gcp/fluentd-gcp-image/fluent.conf

```diff
@@ -42,11 +39,6 @@
 # the name of the Kubernetes container regardless of how many times the
 # Kubernetes pod has been restarted (resulting in a several Docker container IDs).
 
-# Do not directly collect fluentd's own logs to avoid infinite loops.
```

Comment:
No we don't remove them, I'm planning to make a meaningful change to this logic in the separate commit.

For now want to remove unused part in the config.